src/test/java/FindEarliestFreeTimesTest.java

```diff
@@ -0,0 +1,37 @@
+import Commands.Command;
+import Interface.*;
+import Tasks.*;
+import org.junit.jupiter.api.Test;
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.Calendar;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class FindEarliestFreeTimesTest {
+    Ui ui = new Ui();
+    Storage storage = new Storage();
+
+    @Test
+    public void FindEarliestFreeTimesTestEmptyList() throws Exception {
+        String duration = "2";
+        Date date = new Date();
+        DateFormat format = new SimpleDateFormat("E dd/MM/yyyy hh:mm a");
+        Calendar calendar = Calendar.getInstance();
+        calendar.setTime(date);
+        calendar.add(Calendar.HOUR, Integer.parseInt(duration));
+        Date outDate = calendar.getTime();
+        String startDate = format.format(date);
```

Comment:
you can move some of this code to setup and teardown methods. check `@before` and `@after` annotations